src/Env/Config/Manala.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Env\Config;
+
+/**
+ * Make.
```

Comment:
@ogizanagi: The biggest troll ever 😆 

It corresponds to the unix`make` utility, same as `Vagrant` for `vagrant` and `Ansible` for `ansible`. But you're right it's misleading, I'll expand this description